src/actions/dataset.ts

```diff
@@ -48,4 +49,23 @@ export function datasetUrlLoad(name: string, url: string): DatasetUrlLoad {
       }
     );
   };
-}
+};
+
+export const DATASET_RECEIVE = 'DATASET_RECEIVE';
```

Comment:
We could, though i've been wondering if we should unify the `DATASET_URL_RECEIVE` and this? It has not been clear to me what the app does with either the URL or the original data (in the case of inline data), it puts it in the store, but i was under the impression that the schema also encapsulates the data? 

If we unified the two then the action creator for urlReceive would just build a schema and dispatch this single action. 

What do you think?